conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
```

Comment:
This file enables the use of production database for tests.  One thing that needs to confirmed: is this the right place for this file?

Reply:
OK. I can look into this also. We should mainly just configure database access for tests that go in the tests/regression path. The "unit" tests should be completely database free, and the "integration" tests can use a short term database that gets temporarily set up and then torn down after the tests complete.